Client/src/Pages/CreateNewMonitor/index.jsx

```diff
@@ -228,7 +257,14 @@ const CreateNewMonitor = () => {
         rightLayout={
           <div className="advanced-setting-config">
             <div className="adv-setting-title">Check frequency</div>
-            <FlexibileTextField />
+            <div className="monitors-gaps-small"></div>
+            <Dropdown
+              id="freq-dropdown"
+              label="1 minutes"
+              onChange={handleFrequencyOptionChange}
+              options={frequencyOptions}
+              value={frequencyOptions}
```

Comment:
I think we can use the `Select` component instead of `Autocomplete` as users should be picking from a list.  That should resolve some of the errors here.